src/gcpsample_client/manager_initialize.py

```diff
@@ -40,9 +40,7 @@ def displayName(self):
 # manager = managerFactory.createManager('google.manager.gcpsample_asset_manager')
 
 # GF: We're going to externalize which manager we want to use in a toml file and just call this one method once
-manager = ManagerFactory.defaultManagerForInterface("manager.toml", host_interface, factory_impl, logger)
-# GF: Still to verify but I think we don't have to bother calling "initialize" when called this way
-# GF: TODO: Test if we can use settings in the .toml and whether they come through during the initialize
+manager = ManagerFactory.defaultManagerForInterface(host_interface, factory_impl, logger)
```

Comment:
Most hosts fall back on the [environment variable for the path to this file](https://openassetio.github.io/OpenAssetIO/runtime_configuration.html#host_default_config), as it simplifies deployment. i.e `OPENASSETIO_DEFAULT_CONFIG` It's fine to do this if you don't have the ability to set any environment for this host, or you deliberately want it to be working-directory specific with that exact file.